docker-storage-setup.conf

```diff
@@ -84,43 +85,30 @@ DEVICE_WAIT_TIMEOUT=60
 # as this means if there was any leftover data on disk, it will be lost.
 WIPE_SIGNATURES=false
 
-# By default no new volume and filesystem will be setup for docker
-# root dir. Docker creates /var/lib/docker/ on top of underlying filesystem
-# for storing images and containers.
+# By default no new volume and filesystem will be setup for container runtime
+# root dir. For example docker creates /var/lib/docker/ on top of underlying
+# filesystem for storing images and containers.
 #
```

Comment:
can we get rid of this extra line. Otherwise it feels description of new option has started. Right now convention is that new line is there only if we moved onto description of next option.